protocol/x/clob/keeper/equity_tier_limit.go

```diff
@@ -162,24 +160,6 @@ func (k Keeper) ValidateSubaccountEquityTierLimitForStatefulOrder(ctx sdk.Contex
 	}
 
 	equityTierCount := k.GetStatefulOrderCount(ctx, order.OrderId.SubaccountId)
-	// If this is `CheckTx` then we must also add the number of uncommitted stateful orders that this validator
-	// is aware of (orders that are part of the mempool but have yet to proposed in a block).
-	if !lib.IsDeliverTxMode(ctx) {
```

Comment:
just to make sure - we still have the onchain equity tier limit check in deliver tx, right?